app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,47 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+section {
+  overflow: auto;
+}
+.center {
+  text-align: center;
+  vertical-align: center;
+}
+.center h1 {
+	margin-top: 180px;
+  margin-bottom: 30px;
+  color: #000000;
+}
+.jumbotron{
+	background: url(eventhero.jpg);
+	background-size: 100%;
+	height: 100%;
+	margin-bottom: 0px;
+	a{
+	margin-bottom: 150px;
+	}
+	h1{
+		font-size: 80px;
+		font-family: Georgia;
+		font-style: italic;
+		font-weight: bold;
+		color: #fff, #000, #111;
+	}
+}
+.navbar-brand {
+	color: #fff;
+	font-size: 30px;
+	font-family: Georgia;
+}
+.plink{
+	font-size: 20px;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered font-family, font-size